packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -351,15 +367,25 @@ export class SummaryGenerator {
 
             // Update for success/failure
             const ackNackDuration = Date.now() - this.heuristicData.lastAttempt.summaryTime;
-            const telemetryProps: Record<string, number> = {
+
+            // removing 2 irrelevant properties during generate
```

Comment:
Actually, now that I think about, you can let them be in the summarize end event too. It doesn't hurt and its kind of part of the whole summarize process.